apache/superset
# fix: Fixed spacing in alert modal

### SUMMARY

This PR fixes the spacing under the "Message content" section of the alert modal.

Note: When I adjusted the spacing I noticed a few spots in the component that needed translating/cleanup so I added that to this PR as well.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

#### BEFORE:
![alertReportBefore](https://user-images.githubusercontent.com/55605634/200657342-b04b0ea6-6c76-450c-8cb0-3afdc84e8692.png)

#### AFTER:
![alertReportAfter](https://user-images.githubusercontent.com/55605634/200657374-e69915cc-4a77-463c-8182-575cfc754865.png)

#### AFTER (extended)
![alertReportAfterExtendedEdition](https://user-images.githubusercontent.com/55605634/200657453-d4670905-56a3-41a1-8a67-4e8a9d08ca1f.png)


### TESTING INSTRUCTIONS

- Go to the Alerts & reports page and click the "+ ALERT" button
- Observe the spacing between elements

### ADDITIONAL INFORMATION
